### ISSUE TITLE
Is Supports both Gitlab.com and Gitlab CE/EE servers?

### ISSUE THREAD

**Issue by kjkandrea** [nhat-phan/merge-request-integration#99]
I use Gitlab CE/EE, but authentication via token fails.
Gitlab.com has no problems with authentication.

<img width="723" alt="스크린샷 2020-09-23 오후 3 30 10" src="https://user-images.githubusercontent.com/32591477/93975157-ed454500-fdb1-11ea-9ed1-f8eb4cd8a5e3.png">

**Comment by nhat-phan** [comment]
yes, it should work both gitlab.com and Gitlab CE/EE. Is your server run under a VPN with self generated credential?
